mysql-test/suite/innodb_fts/t/sync.test

```diff
@@ -168,3 +168,1190 @@ SELECT * FROM t1 WHERE MATCH(title) AGAINST ('mysql database good');
 DROP TABLE t1;
 
 --source include/wait_until_count_sessions.inc
```

Comment:
The new testcase - if it stays in this file - should probably go above this line